packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -41,31 +42,28 @@ export function RenderConfig({
     }
     config = result.data
   }
+  const configPath = 'workflow'
   const FrameWrapper = getComponent(
     config.frameWrapper ?? 'OTDefaultFrameWrapper',
-    'workflow',
+    configPath,
     COMPONENTS,
   ) as FrameWrapper
   const globalContext: GlobalContext = {
     config,
     COMPONENTS,
     FrameWrapper,
   }
+  // Workflows are just frames! Use unknown to convince TS of this fact.
+  const frame = {
+    ...config,
+    view: { component: '__FRAGMENT__' },
```

Comment:
Not sure about this hack yet. Need something to say this is top level and it's okay to not have a component

Reply:
Hacks FTW